src/vm/win32threadpool.cpp

```diff
@@ -1815,8 +1815,8 @@ Thread* ThreadpoolMgr::CreateUnimpersonatedThread(LPTHREAD_START_ROUTINE lpStart
         // CreateNewThread takes care of reverting any impersonation - so dont do anything here.
         bOK = pThread->CreateNewThread(0,               // default stack size
                                        lpStartAddress,
-                                       lpArgs           //arguments
-                                       );
+                                       lpArgs,           //arguments
+                                       W("ThreadPool"));
```

Comment:
Can we compromise on `.NET ThreadPool`. I do think the name will be helpful since there are often several threadpools and others may choose the name `ThreadPool`

@jkotas I assume one can't run .NET Core and .NET Framework concurrently in the same process.

/cc @stephentoub